src/main/java/Employee.java

```diff
@@ -0,0 +1,52 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Employee {
```

Comment:
this is violation of single responsibility principle. 
leave employee fields, constructor, getter, setters.
and move all the rest to a new utility class, Employees